mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -265,6 +266,7 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 	METHOD method_35054 setNoDrag (Z)V
 		ARG 1 noDrag
 	METHOD method_36362 updateGlowing ()V
+	METHOD method_36549 addDeathParticles ()V
```

Comment:
Should this be referred to "death" or "poof" particles?

Reply:
I originally had it as Poof but changed to Death... if people want me to change it back, sure

Reply:
sure, death is more straightforward for people who call it, though i wonder if people do call it... You can just keep it.

Reply:
Death particles is good.